### ISSUE TITLE
groups (and classrooms)

### ISSUE THREAD

**Issue by fjenett** [nodebox/live#8]
It would be great to have a way of grouping users, so that one can more easily get an overview, find their projects, etc.

In regards to classrooms it would be great to have a way to also group projects. This would allow for making sets for workshops or as results of an assignment.

**Comment by craftoid** [comment]
Agreed! I am currently using the background color of the titles, to get some kind of order into my projects. It would already be great if they could be sorted by color or title.
